.github/pull_request_template.md

```diff
@@ -8,6 +8,7 @@
 - `yarn test:unit:all` should run without errors or warnings
 - `yarn serve` should run without errors or warnings
```

Comment:
* The search box seems to go off of the main part of the page. Intentional, or is this just a temp design? The page also "jumps" when I open and close the search modal.
<img width="1426" alt="image" src="https://github.com/user-attachments/assets/b726bbbf-b2aa-4a18-b732-5690b0ee644d">

* Similarly on mobile...
<img width="278" alt="image" src="https://github.com/user-attachments/assets/d3a5afce-2a3f-478f-b7dc-d314615b6ff5">

Reply:
I think was was the lo-fi design at the time, though those seem to be hidden in Figma now. The hi-fi design calls for a UX that is slightly different, so we'll be touching this again when the hi-fi implementation for this screen happens. If you want to request temp changes to the current lo-fi implementation, we can create a ticket.

Reply:
No need, especially if we're changing soon. I expect as we move more and more towards the vertical slice model, we'll have fewer of these things come up anyways.